tailwind.config.ts

```
@@ -0,0 +1,148 @@
+        'fade-out': 'fade-out 0.4s ease-out',
+        'scale-in': 'scale-in 0.3s ease-out',
+        'scale-out': 'scale-out 0.3s ease-out',
+        'slide-in-right': 'slide-in-right 0.4s ease-out',
+        'slide-out-right': 'slide-out-right 0.4s ease-out',
+        'enter': 'fade-in 0.4s ease-out, scale-in 0.3s ease-out',
+        'exit': 'fade-out 0.4s ease-out, scale-out 0.3s ease-out',
+        'pulse-slow': 'pulse-slow 2s ease-in-out infinite'
+      }
+    }
+  },
+  plugins: [require("tailwindcss-animate")],
```

ESLint

[error] A `require()` style import is forbidden.

(@typescript-eslint/no-require-imports)